vllm/compilation/pass_manager.py

```diff
@@ -64,6 +65,7 @@ def configure(self, config: VllmConfig):
         if self.pass_config.enable_fusion:
             self.passes += [FusionPass.instance(config)]
             self.passes += [ActivationQuantFusionPass(config)]
```

Comment:
Not needed for this PR: In next PR, we should rename this to `ActivationFP8QuantFusionPass`